src/services/tracing.ts

```diff
@@ -278,3 +278,4 @@ export const addAttributesToCurrentSpanAndPropagate = <F extends () => ReturnTyp
 export { SemanticAttributes, SemanticResourceAttributes }
 
 export const ENDUSER_ALIAS = "enduser.alias"
+export const ENDACCOUNT_DEFAULTWALLETID = "endaccount.defaultWalletId"
```

Comment:
I don't understand the prefix `end` here.
Also I would say we should be recording the `targetAccount.id` and / or `targetWallet.id` or something like that rather than `defaultWalletId` - it seems a bit indirect.

Reply:
> I don't understand the prefix `end` here.

Sounded strange to me as well, but left it in to keep the `enduser` context and to see if anyone else noticed it was odd.

> Also I would say we should be recording the `targetAccount.id` and / or `targetWallet.id` or something like that rather than `defaultWalletId` - it seems a bit indirect.

The original idea here was to record something that we had access to at the graphql layer, and my 1st thought was walletId since we include that as input in a lot of requests. Thinking about it again though, the [uuid-representation of the userId](https://github.com/GaloyMoney/galoy/blob/70cba538e035acdc0af33ca84fc143208ee76127/src/graphql/types/object/graphql-user.ts#L26) could work just as well for this. I'm trying to find something that's non-internal to record.

Reply:
what is the goal of `getUuidByString(source.id)` for the user.id? not exposing the user.id? (this is what is in the token so it's exposed anyway currently)

Reply:
> what is the goal of `getUuidByString(source.id)` for the user.id? not exposing the user.id? (this is what is in the token so it's exposed anyway currently)

Being able to quickly move from something like a `me` query to being able to trace a user's actions through honeycomb without having to necessarily decode the auth token or lookup the user in the database. It's a very specific use-case, so I'm ok either way if we decide to include or not, or if there is a better way to quickly find a given user in honeycomb from easily accessible info on the user.

Reply:
but the `getUuidByString(source.id)` is also something that is exposed to the user directly right? do you make any use of it @daviroo ?

Reply:
> that is exposed to the user directly right

I'm not sure if we present it in the UI anywhere, but it does come back on a `me` query